xchange-okcoin/src/main/java/com/xeiam/xchange/okcoin/service/polling/account/OkCoinFuturesAccountService.java

```diff
@@ -1,4 +1,4 @@
-package com.xeiam.xchange.okcoin.service.polling;
+package com.xeiam.xchange.okcoin.service.polling.account;
```

Comment:
I would keep it in the original package, i.e. pooling. Otherwise it brings incompatibility with the older version. I would especially avoid doing this change in 'pooling' since our work is related to 'streaming'